src/components/Auth/LandingBody.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, memo } from 'react';
+import { Button, IconButton } from '@material-ui/core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faGithub } from '@fortawesome/free-brands-svg-icons';
+import { faEnvelope } from '@fortawesome/free-solid-svg-icons';
+import clsx from 'clsx';
+
+import example from 'assets/example-pathway.png';
+import useStyles from './styles';
+
+const LandingBody: FC = () => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.landingBody}>
+      <div className={styles.landingContent}>
+        <div className={clsx(styles.text, styles.mainText)}>
+          A simple app to build customized clinical pathways
+        </div>
+
+        <div className={clsx(styles.text, styles.subText)}>
+          using{' '}
+          <a
+            href="https://mcodeinitiative.org/"
+            className={styles.link}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            mCODE
+          </a>
+          , a standardized data model around cancer
+        </div>
+
+        <Button className={styles.tryItButton}>Try It</Button>
```

Comment:
Can we have this redirect to `/demo`. I know its technically part of the other task but just so we have a way to get to the actual builder.

Reply:
I know see your comment above, I guess there is a way so its up to you whether to implement this here or just let PATHWAYS-389 implement it

Reply:
I think it's a good idea, so I implemented it. :)